src/components/Elements/NodeList.js

```diff
@@ -1,26 +1,37 @@
 import React, { Component } from 'react';
 import { Node } from '../Elements';
-import Droppable from '../../containers/Elements/Droppable';
+import { scrollable, droppable, draggable, selectable } from '../../behaviors';
+
+const EnhancedNode = draggable(selectable(Node));
 
 class NodeList extends Component {
   render() {
     const {
-      network: {
-        nodes
-      }
+      network,
+      label,
+      isActive,
+      activeNodeAttributes,
+      handleSelectNode,
+      handleDropNode,
+      draggableType,
     } = this.props;
 
     return (
-      <Droppable name="nodelist">
-        <div className='node-list'>
-          { nodes.map((node, index) => {
-            const label = `${node.nickname}`;
-            return <Node key={ index } label={ label } />;
-          }) }
-        </div>
-      </Droppable>
+      <div className='node-list'>
+        { network.nodes.map((node, index) => {
+          return <EnhancedNode key={ index } label={ label(node) } isActive={ isActive(node) } activeNodeAttributes={ activeNodeAttributes } onSelected={ () => handleSelectNode(node) } onDropped={ (hits) => handleDropNode(hits, node) } draggableType={ draggableType } { ...node } />;
```

Comment:
Do we have length of line being linted? This is a little hard to read all in one line.